pkg/operator/ceph/file/mds/mds_liveness_probe.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+ROOK_CEPH_MON_HOST="{{ .RookCephMonHost }}"
+ROOK_CEPH_MON_INITIAL_MEMBERS="{{ .RookCephMonInitialMembers }}"
+DAEMON_MDS="{{ .DaemonMds }}"
+FILESYSTEM_NAME="{{ .FilesystemName }}"
+
+outp="$(ceph fs dump --mon-host="$ROOK_CEPH_MON_HOST" --mon-initial-members="$ROOK_CEPH_MON_INITIAL_MEMBERS" --format json 2>&1)"
+rc=$?
+if [ $rc -ne 0 ]; then
+    echo "ceph mds dump check failed with the following output:"
+    echo "$outp" | sed -e 's/^/> /g'
+    echo "we couldn't get the information to reliably tell if the MDS should be restarted or not"
+    return 0
+fi
+
+# get the active and standby mds in the fs map
+standbyMds=$(echo "$outp" | jq ".standbys|map(.name)")
+activeMds=$(ceph fs get "$FILESYSTEM_NAME" --mon-host="$ROOK_CEPH_MON_HOST" --mon-initial-members="$ROOK_CEPH_MON_INITIAL_MEMBERS" --format json 2>&1 | jq ".mdsmap.info|map(.name)")
```

Comment:
Why `2>&1` here?

Normally, stderr (2) is not captured from the subshell. This should mean that stdout (&1) will have either valid json or no output.

This seems like good behavior to me, but I might be missing something.

Reply:
right, any other way you would suggest?